server/handlers/database_handlers.go

```diff
@@ -100,3 +105,76 @@ func (h *Handler) GetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *m
 	}
 	fmt.Fprint(w, string(val))
 }
+
+func (h *Handler) ResetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *models.Preference, _ *models.User, provider models.Provider) {
+
+	mesherydbPath := path.Join(utils.GetHome(), ".meshery/config")
+	err := os.Mkdir(path.Join(mesherydbPath, ".archive"), os.ModePerm)
+	if err != nil && os.IsNotExist(err) {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+	src := path.Join(mesherydbPath, "mesherydb.sql")
+	dst := path.Join(mesherydbPath, ".archive/mesherydb.sql")
+
+	fin, err := os.Open(src)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+	defer fin.Close()
+
+	fout, err := os.Create(dst)
```

Comment:
What if `.archive/mesherydb.sql` already exists? maybe attach a timestamp or something on the filename.

Reply:
Done, thanks.